README.md

```diff
@@ -2,31 +2,58 @@
 
 Implementation of a [TileSource](https://openseadragon.github.io/docs/OpenSeadragon.TileSource.html) for [OpenSeadragon](https://openseadragon.github.io/) based on [geotiff.js](https://geotiffjs.github.io/), enabling local and remote TIFF files to be viewed without using an image server.
 
+Implements an ES6 module extending the original work from [pearcetm/GeoTIFFTileSource](https://github.com/pearcetm/GeoTIFFTileSource). For further information on creating GeoTIFF files and it's requirements, please see the original repository.
```

Comment:
This note only makes sense in the fork. In order to merge, this should be removed or updated to make sense in the GeoTIFFTileSource repo.

Reply:
Noted, will remove this